packages/graph/schema.graphql

```diff
@@ -85,4 +86,33 @@ type RoundProject @entity {
   metaPtr: MetaPtr!
   round: Round!
   payoutAddress: String
+}
+
+
+########################
+# = VotingStrategy = #
+########################
+
+# id = votingContractAddress.toHex()
+type VotingStrategy @entity {
+  id: ID!
+  strategyName: String!
+  strategyAddress: String!
+  round: Round @derivedFrom(field: "votingStrategy")
+  votes: [QFVote!] @derivedFrom(field: "votingStrategy")
+}
+
+########################
+# = QFVotingStrategy = #
+########################
+
+# id = keccak256(event.receipt.transactionHash.toHex(), QFVote.to)
+type QFVote @entity {
+  id: ID!
+  votingStrategy: VotingStrategy!
+  token: String!
+  amount: BigInt!
+  from: String!
+  to: String!
+  # round: Round!
```

Comment:
I would love to integrate this but it looks like since VotingStrategy.Round is not set when the data is indexed , this fails when you try setting it via votingStrategy.round

Here is code which was removed https://github.com/gitcoinco/grants-round/pull/532/commits/a87a2b147e5cf85f1a7e6b09791a07fc77aaf5f3#r1015412397

Reply:
This may be due to the ordering 

> The following process is used to order the triggers for a data source within a block:
> 
> - The transaction index within the block orders event and call triggers
> - Should there be event and call triggers within the same transaction, the following convention is used for ordering: event triggers first then call triggers. Each type respects the particular order they are defined in the manifest.
> - After event and call triggers, block triggers are run in the order as defined in the manifest.